### PR TITLE
feat: reference crowdin/cli directly in action.yml `runs.image`

### PR DESCRIPTION
**Pull request by iamstarkov**
you dont need to have extra dockerfile, you can reference needed image directly and this way dockerfile isn't build on every run, which makes builds slower and less optimal than expected

```
runs:
  using: 'docker'
  image: 'docker://debian:stretch-slim'
```

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-using-public-docker-registry-container

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsimage

**Important: additional code setup like apk and entry point needs to be moved to a base crowdin image** 

https://github.com/crowdin/github-action/blob/master/Dockerfile#L3C1-L8C30

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -194,4 +194,4 @@ outputs:
 
 runs:
   using: docker
-  image: 'Dockerfile'
+  image: 'docker://crowdin/cli:4.1.0'
```
Comment (iamstarkov) [inline action.yml:197]:
```suggestion
  image: 'docker://crowdin/cli:4.1.0'
  entrypoint: 'entrypoint.sh'
```

**Comment by andrii-bodnar** [comment]
Unfortunately, it doesn't work that way. See the logs - https://github.com/crowdin/github-action/actions/runs/10060485921/job/27808235845

Also, the Dockerfile installs the required tools on which the action depends.

```
RUN apk --no-cache add curl git git-lfs jq gnupg;
```

**Comment by iamstarkov** [comment]
let me fix that really quick

**Comment by iamstarkov** [comment]
@andrii-bodnar fixed https://github.com/crowdin/crowdin-cli/pull/827

**Pull request closed without merging**